proto/pocket/migration/tx.proto

```diff
@@ -187,6 +192,9 @@ message MsgClaimMorseSupplier {
   // See: https://dev.poktroll.com/operate/configs/supplier_staking_config#staking-types.
   string shannon_operator_address = 2 [(cosmos_proto.scalar) = "cosmos.AddressString", (gogoproto.jsontag) = "shannon_operator_address"];
 
+  // The ed25519 public key of the morse account with morse_src_address.
+  bytes morse_public_key = 6 [(gogoproto.casttype) = "crypto/ed25519.PublicKey"];
```

Comment:
Can we remove `morse_src_address` if we have this?

Reply:
See https://github.com/pokt-network/poktroll/pull/1179#discussion_r2039511580